hadoop-hdds/client/src/main/java/org/apache/hadoop/hdds/scm/OzoneClientConfig.java

```diff
@@ -54,6 +54,18 @@
       tags = ConfigTag.CLIENT)
   private int streamBufferSize = 4 * 1024 * 1024;
 
+  @Config(key = "stream.buffer.increment",
+      defaultValue = "0",
```

Comment:
```suggestion
      defaultValue = "0B",
```

(Unfortunately "0" is not valid for `StorageSize`).